src/reaper_osara.cpp

```diff
@@ -331,6 +331,31 @@ bool isItemSelected(MediaItem* item) {
 	return *(bool*)GetSetMediaItemInfo(item, "B_UISEL", NULL);
 }
 
+char* automationModeAsString(int mode) {
+	// this works for track automation mode and global automation override.
+	switch (mode) {
+	case -1:
+		return "no override";
+	case 0:
+		return "Trim/Read";
```

Comment:
Some of these messages are capitalised, others aren't. I'd prefer they were consistent. I guess lower case makes sense, since they're being appended to the end of a message.

Reply:
For some reason eloquence says "trim/red" if it is lower case.